ui/src/components/Document/DocumentEdition.js

```diff
@@ -1,5 +1,16 @@
 import React from 'react';
+import { Item } from 'semantic-ui-react';
 
-export const DocumentEdition = ({ document }) => {
-  return document.metadata.edition ? `ed.  ${document.metadata.edition}, ` : '';
+export const DocumentEdition = ({ document, explicit }) => {
+  if (explicit) {
```

Comment:
I am not sure I understand this `explicit` param, why is that needed?

Reply:
to display the label, I renamed it